Snippets/Snippets_5/Encryption/Configuration/encryption.config.xml

```diff
@@ -1,15 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
-  <!-- startcode EncryptionFromAppConfig-->
+  <!-- startcode EncryptionFromAppConfig -->
   <configSections>
-    <section name="RijndaelEncryptionServiceConfig"
-             type="NServiceBus.Config.RijndaelEncryptionServiceConfig, NServiceBus.Core"/>
+    <section
+      name="RijndaelEncryptionServiceConfig"
+      type="NServiceBus.Config.RijndaelEncryptionServiceConfig, NServiceBus.Core"/>
   </configSections>
-  <RijndaelEncryptionServiceConfig Key="gdDbqRpQdRbTs3mhdZh9qCaDaxJXl+e6">
+  <RijndaelEncryptionServiceConfig Key="tWIrVnPy3Smd2r/xcSb9r/1wcVY9/3KWbrzIaemx5yw=" KeyIdentifier="B" KeyFormat="Base64"
```

Comment:
since this will render inline can we newline the attributes to reduce dodgy wrapping in the html?